index.html

```diff
@@ -1,239 +1,313 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viindewport" content="width=device-width, initial-scale=1.0" />
-    <title>Alice's Portfolio - Full Stack Developer</title>
-    <link rel="stylesheet" href="style.css" />
-    <link
-      href="https://fonts.googleapis.com/css?family=Playfair Display"
-      rel="stylesheet"
-    />
-    <link
-      href="https://fonts.googleapis.com/css?family=Poppins"
-      rel="stylesheet"
-    />
-    <link
-      rel="stylesheet"
-      href="https://fonts.googleapis.com/icon?family=Material+Icons"
-    />
+	<meta charset="UTF-8" />
+	<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+	<meta name="viindewport" content="width=device-width, initial-scale=1.0" />
+	<title>Alice's Portfolio - Full Stack Developer</title>
+	<link rel="stylesheet" href="style.css" />
+	<link
+  	href="https://fonts.googleapis.com/css?family=Playfair Display"
+  	rel="stylesheet"
+	/>
+	<link
+  	href="https://fonts.googleapis.com/css?family=Poppins"
+  	rel="stylesheet"
+	/>
+	<link
+  	rel="stylesheet"
+  	href="https://fonts.googleapis.com/icon?family=Material+Icons"
+	/>
+	<script src="https://kit.fontawesome.com/4905cce8d8.js" crossorigin="anonymous"></script>
   </head>
   <body>
-    <header class="header">
-      <nav class="headline">
-        <div class="desktopmenu">
-          <ul>
-            <li><a href="#hello"> Hello</a></li>
-            <li><a href="#portfolio-section"> Portfolio</a></li>
-            <li><a href="#about-section"> About</a></li>
-            <li><a href="#contact"> Contact</a></li>
-          </ul>
-        </div>
-        <a href="#" class="icon"></a>
-        <a href="#" class="menu"></a>
-      </nav>
-    </header>
-  <header>
-    <section class="headline-section"> 
-      <div class="info-area"> 
-        <h1 class="my-name">Alice Mirigo</h1>
-        <p class="detail">
-          Hello! I am a software developer! I can help you build a product,
-          feature or website. Taka a look of my works. If you like what you see
-          and have a project you need coded, don’t hesitate and contact me.
-        </p>
-        <button type="button" class="head-button">Start Collaboration</button>
-        <img id="dot-image" src="./images/image_dots.svg" alt="Geo background">
-      </div>
-    </section>
-      <img class="geo-image" src="./images/image-geometry_1.svg" alt="Project image" />
-  </header>
-<section>
-  <div class="worksection"><img src="./images/Group 117.svg" alt="Project Image"></div>
-</section>
+	<header class="header">
+    	<nav class="headline">
+      	<i id="hamburger" class="hamburger fas fa-bars fa-lg "></i>
```

Comment:
Great job team 🤝

- [ ] [Optional] Kindly remove the menu icon from your mobile menu. This will make your design similar to the one given in the Figma template.
![mobile-menu-image](https://user-images.githubusercontent.com/63917725/210656137-7c19ee98-8544-4f73-b08f-802da4a91e58.PNG)